lib/widgets/back_button.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import '/shared/constants.dart';
+
+class MyBackButton extends StatelessWidget {
+  const MyBackButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton(
+      backgroundColor: ConstantsClass.appColor,
+      onPressed: () {
+        Navigator.pop(context);
+      },
+      heroTag: null,
```

Comment:
unnecesary parameter, heroTag is null by default

Reply:
I have two FloatingActionButtons in the same route, so I need to specify heroTag:null to avoid conflicts (shouldn't share same tag)